### PR TITLE
updates

### PR DESCRIPTION
**Pull request by eprince1**
expose to static sites

### CODE REVIEW

**Review by avisprince — CHANGES_REQUESTED** [review]
module.js needs to be part of gitignore

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "jellysync",
-  "version": "1.0.1",
+  "version": "1.0.6",
```
Comment (avisprince) [inline package.json:3]: why the change?

**Review by eprince1 — COMMENTED** [review]
Not sure if it's needed for upload so that the cdn can access it

**Review by avisprince — COMMENTED** [review]
does it upload directly from github?

File: package.json (hunk 1 above)
Comment (eprince1) [reply package.json:3]: U can never publish the same version to npm
